test/integration/default/serverspec/default_spec.rb

```diff
@@ -13,6 +13,15 @@
     it { should exist }
   end
 
+  describe file('/root/backup_minecraft.sh') do
+    its(:content) { should match /$DATE-backup-minecraft.tar.gz/ }
+    it { should exist }
+  end
+
+  describe cron do
+    it { should have_entry '00 02 * * * /root/backup_minecraft.sh' }
```

Comment:
You should stub the node attributes here, or else hard-coded the cron settings in the recipe.